### DISCUSSION TITLE
Caching Header for API Responses

### DISCUSSION THREAD

**Discussion by marcus-at-localhost** [atrocore/atrocore#179] [Ideas]
Hi is it possible to send ETag header and/or last modified headers with the api responses?
I'm consuming the api with guzzle and for an entity I need to query up to 5 endpoints and I could force caching, but then I have to force clean the cache if something changes or if the cache gets stale.

Thanks for considering.

**Comment by rratsun** [comment]
Hello

Almost all entities have "modifiedAt" field, so you can get this data from the response body.
Here is an example getting the collection of products with "modifiedAt" field:
GET https://demo.atropim.com/api/v1/Product?select=name%2CmodifiedAt
